server/sonar-web/src/main/js/apps/component-measures/components/bubbleChart/BubbleChart.js

```diff
@@ -106,6 +106,7 @@ export default class BubbleChart extends React.Component {
   }
 
   getTooltip(component) {
+    /* eslint-disable max-len */
```

Comment:
Its disabling the rule for the rest of the file, maybe we should re-enable it after the array definition.